models/base/scratch/default/snowplow_web_base_events_this_run.sql

```diff
@@ -6,313 +6,68 @@
   )
 }}
 
-{%- set lower_limit, upper_limit = snowplow_utils.return_limits_from_model(ref('snowplow_web_base_sessions_this_run'),
-                                                                          'start_tstamp',
-                                                                          'end_tstamp') %}
+{% set contexts = var('snowplow__entities_or_sdes', []).copy() %}
```

Comment:
I would document why you do the copy here before we forget about your workaround